### PR TITLE
Updated Windows CI to download the CLI correctly.

### PR DESCRIPTION
**Pull request by armandgrillet**

### CODE REVIEW

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

File: scripts/download_dcos_windows.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import ssl
+import sys
+import urllib.request
+
+if len(sys.argv) != 2:
+    raise Exception("This scrips needs one argument to know where to download dcos.exe")
```
Comment (rgo3) [inline scripts/download_dcos_windows.py]: Typo: "This script.."

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

File: ci/integration-tests-windows.groovy [hunk 2]
```diff
@@ -86,8 +89,7 @@ node('py36') {
                                 python scripts/plugin/package_plugin.py; \
                                 cd python/lib/dcoscli; \
                                 make env; \
-                                rm -f ./env/Scripts/dcos.exe; \
-                                python -c 'import urllib.request; urllib.request.urlretrieve(\\\"https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe\\\", \\\"dist/dcos.exe\\\")'; \
+                                mv ../../../dcos.exe dist; \
```
Comment (bamarni) [inline ci/integration-tests-windows.groovy:92]: Did you try to run `wget` directly from this node? It has winbash installed and they might have `wget` there.

**Review by bamarni — APPROVED** [review]
one question otherwise lgtm

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

File: ci/integration-tests-windows.groovy (hunk 2 above)
Comment (armandgrillet) [reply ci/integration-tests-windows.groovy:92]:
```
+ wget -O dist/dcos.exe https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe
--17:41:18--  https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe
           => `dist/dcos.exe'
Resolving downloads.dcos.io... done.
Connecting to downloads.dcos.io[99.84.239.15]:443... connected.

Unable to establish SSL connection.

Unable to establish SSL connection.
```

**Pull request merged**